content/1_General/Debugging_General.mdx

```diff
@@ -22,13 +22,25 @@ This module is based on both of these resources. I've included the content that
 
 ## Pre-Submit
 
-<IncompleteSection>
-
-todo - find particularly bad example of repetition
-
-</IncompleteSection>
-
  - Don't repeat yourself!
+   - For example, if you find yourself rewriting the same parts of your code over and over again, consider creating a function you can use instead.
+   ```
+   public static void main(String[] args) throws IOException {
+           int n = Integer.parseInt(stringTokenizer.nextToken());
+           int k = Integer.parseInt(stringTokenizer.nextToken());
+   }
+   ```
+   can be simplified to...
+   ```
+   private static int nextInt() {
+   	Integer.parseInt(stringTokenizer.nextToken());
+   }
+   
+   public static void main(String[] args) throws IOException {
+   	int n = nextInt();
+	int k = nextInt();
+   }
```

Comment:
idt IO is a good example for this maybe like doing some operations??

Reply:
Ok so like maybe swapping items in an array?